Karl/integrations/nodejs/test.js

```diff
@@ -83,8 +83,44 @@ describe('Claire JS client test.', function () {
     checkKarlIsUp();
   });
 
-  it('Spawn Claire instance and register in Karl', function (done) {
+  it('Spawn Claire instance and test adding/removing dependencies', function (done) {
     claire = new Claire(claireConfig);
+    var completed = false;
+
+    claire.addDependency('http://localhost:8283', function (error) {
+      assert(!error);
+      assert.equal(claire.dependencies.length, 1);
+      assert.equal(claire.dependencies[0].ip, '127.0.0.1');
+      assert.equal(claire.dependencies[0].port, 8283);
+      assert.equal(claire.dependencies[0].prefix, '/');
+      claire.addDependency('http://localhost:12345', function (error) {
+        assert(!error);
+        assert.equal(claire.dependencies.length, 2);
+        assert.equal(claire.dependencies[1].ip, '127.0.0.1');
+        assert.equal(claire.dependencies[1].port, 12345);
+        assert.equal(claire.dependencies[1].prefix, '/');
+        claire.removeDependency('http://localhost:12345', function (error) {
+          assert(!error);
+          assert.equal(claire.dependencies.length, 1);
+          assert.equal(claire.dependencies[0].ip, '127.0.0.1');
+          assert.equal(claire.dependencies[0].port, 8283);
+          assert.equal(claire.dependencies[0].prefix, '/');
+          completed = true;
+        });
+      });
+    });
+
+    var wait = function () {
```

Comment:
But the variable name is used here to reference this function in the setTimeout call.